ui/app/components/tx-view.js

```diff
@@ -41,85 +41,103 @@ function TxView () {
 }
 
 TxView.prototype.render = function () {
+  const selected = '0x82df11beb942BEeeD58d466fCb0F0791365C7684' // TODO: remove fake address
+
   return h('div.tx-view.flex-column', {
     style: {
-      flexGrow: 2,
-      flexShrink: 0,
-      flexBasis: '230px',
+      flex: '62 0 62%',
```

Comment:
Does this work for compatibility? I know that when using `transform` inline, you have to do `webkitTransform`.